adobe_vipm/flows/fulfillment/shared.py

```diff
@@ -323,3 +324,29 @@ def add_subscription(
         f'Subscription {item["subscriptionId"]} ({subscription["id"]}) '
         f'created for order {order["id"]}'
     )
+
+
+def set_subscription_actual_sku(
```

Comment:
In case of actions within the cancellation window, the SKU is taken looking up the subscription in adobe so the `sku` keyword argument will be null.
In case of operations outside the cancellation window, the SKU is taken from the PREVIEW_RENEWAL order type in order to be consistent in MPT.